sql/database.sql

```diff
@@ -8559,6 +8559,7 @@ CREATE TABLE ar_activity (
   follow_up_note text,
   account_code varchar(15) NOT NULL,
   reason_code varchar(255) DEFAULT NULL COMMENT 'Use as needed to show the primary payer adjustment reason code',
+  deleted        datetime DEFAULT NULL COMMENT 'NULL if active, otherwise when voided',
```

Comment:
just making sure that zero datetime (00-00-0000 etc.) is never meant to be placed here (project is working toward supporting strict standards)

Reply:
Right, 0000-00-00 would be an error.